addons/mail/models/ir_actions_server.py

```diff
@@ -25,12 +25,14 @@ class IrActionsServer(models.Model):
         tracking=True,
         selection_add=[
             ('next_activity', 'Create Activity'),
+            ('discuss_post', 'Send Message'),
```

Comment:
Please make sure to have everything about channel in the `discuss` subfolder, so everything done in this file should be done as an override in the other folder instead.